src/Actor/You.pm

```diff
@@ -117,6 +122,24 @@ sub inventory {
 }
 
 ##
+# InventoryList $char->storage()
+# Ensures: defined(result)
+#
+# Get the storage list for this character.
+sub storage {
+	return $_[0]->{__storage};
+}
```

Comment:
It was just done to be able to use `$char->storage->`, I don't think directly accessing these vriables is a good practice.